src/main/java/seedu/task/logic/commands/AddCommand.java

```diff
@@ -23,33 +26,48 @@ public class AddCommand extends Command {
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
-            + PREFIX_DESCRIPTION + "Example description"
+            + PREFIX_DESCRIPTION + "Example description "
             + PREFIX_TAG + "friends "
             + PREFIX_TAG + "owesMoney";
 
     public static final String MESSAGE_SUCCESS = "New task added: %1$s";
     public static final String MESSAGE_DUPLICATE_TASK = "This task already exists in the task book";
 
-    private final Task toAdd;
+    private final ArrayList<Task> toAdd;
+
+    /**
+     * Creates an AddCommand to add the specified {@code List<Task>}
+     */
+    public AddCommand(List<Task> taskList) {
+        requireNonNull(taskList);
+        toAdd = new ArrayList<Task>();
+        for (Task cur: taskList) {
+            toAdd.add(cur);
+        }
+    }
 
     /**
      * Creates an AddCommand to add the specified {@code Task}
      */
     public AddCommand(Task task) {
         requireNonNull(task);
-        toAdd = task;
+        toAdd = new ArrayList<Task>();
+        toAdd.add(task);
     }
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
-        if (model.hasTask(toAdd)) {
-            throw new CommandException(MESSAGE_DUPLICATE_TASK);
+        for (Task cur: toAdd) {
+            if (model.hasTask(cur)) {
+                throw new CommandException(MESSAGE_DUPLICATE_TASK);
+            }
         }
-
-        model.addTask(toAdd);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+        for (Task cur: toAdd) {
+            model.addTask(cur);
+        }
+        return new CommandResult(String.format(MESSAGE_SUCCESS, formatAddedTasks(toAdd)));
```

Comment:
Would it be better to combine the 2 for loops together?
An issue might occur when the user tries to add in 2 same commands that is not in the storage